### PR TITLE
fix issues #146, #161, #171

### PR DESCRIPTION
**Pull request by kangmingtay**
- [x] Fix inconsistency in format of commands

- [x] Fix command given in the getting started section of the User Guide does not work

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -275,23 +275,31 @@ Clocks out for the employee John Doe.
 
 ==== Add Order
 
+Important Terminology
+
+* _A customer is anyone who wants to place an order._
+* _A member in the RMS is a returning customer._
```
Comment (px1099) [inline docs/UserGuide.adoc]: Should be changed to:
A member in RMS is a registered customer who enjoys more benefits than a normal customer.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -275,23 +275,31 @@ Clocks out for the employee John Doe.
...
-Display the current draft and the list of order draft commands used for adding a new order +
+Displays the current draft, steps to add an order and the list of order draft commands that can be used +
 +
 Format: `addorder`
 
-* *Pick A Member As The Customer Of The Draft* +
+
+_The following step is only necessary if the customer is a member._
+
+* *Select A Member As The Customer Of The Draft* +
```
Comment (px1099) [inline docs/UserGuide.adoc:300]: Line 300 should be a note, and it should be under line 302

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -275,23 +275,31 @@ Clocks out for the employee John Doe.
 
 ==== Add Order
 
+Important Terminology
```
Comment (px1099) [inline docs/UserGuide.adoc]: This should be in its own section that explains all four terminologies before the list of commands section

**Review by px1099 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (kangmingtay) [reply docs/UserGuide.adoc]: ok that sounds good

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -297,9 +297,8 @@ Displays the current draft, steps to add an order and the list of order draft co
 Format: `addorder`
 
 
-_The following step is only necessary if the customer is a member._
-
 * *Select A Member As The Customer Of The Draft* +
+_The following step is only necessary if the customer is a member._
```
Comment (px1099) [inline docs/UserGuide.adoc]: Change "The following step" to "This step"

**Review by px1099 — CHANGES_REQUESTED** [review]

**Comment by px1099** [comment]
Resolve #146, #161

**Review by px1099 — APPROVED** [review]

**Pull request merged**